jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -89,10 +110,34 @@ object XGBoost extends Serializable {
     } else {
       throw new IllegalArgumentException(
         s"Encountered a partition with $nUndefined NaN base margin values. " +
-            "If you want to specify base margin, ensure all values are non-NaN.")
+          "If you want to specify base margin, ensure all values are non-NaN.")
```

Comment:
Is the rest of the project also using 2 spaces for continuation indent?

Reply:
This was done automatically by my IDE, but yeah, it seems there is two space continuation indent in other places as well.